app/models/new_policy_class.rb

```diff
@@ -22,4 +23,10 @@ def menu
       pluck(:id, :section, :name)
     end
   end
+
+  def all_policies_are_determined
+    return if planning_application_policy_sections.none?(&:to_be_determined?)
+
+    errors.add(:base, :policies_to_be_determined)
+  end
```

Comment:
Is this used? There's a similar method in the `Review` model, but this version of it doesn't seem to get added as a validator unless I'm missing something.

Reply:
No, not used. Good spot 👍